tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -419,7 +428,7 @@ private List<TiRegion> splitRegion(List<ByteString> splitKeys, BackOffer backOff
             "split key equal to region start key or end key. Region splitting is not needed.");
       } else {
         logger.info("start to split region id={}, split size={}", region.getId(), splits.size());
-        List<TiRegion> newRegions;
+        List<Metapb.Region> newRegions;
```

Comment:
Why use TiStore and Metapb.Region rather than Metapb.Store and TiRegion

Reply:
Only here the private method will use MetaPB.Region. The reason is that the construction method of TiRegion is not as simple as before (the old TIRegion can be used directly), I copied the upstream to write the code and the interface of RegionStoreClient is very different upstream and downstream, I will merge the upstream and downstream RegionStoreClient in another pull request.